tests/api/runtimes/test_nuclio.py

```diff
@@ -394,6 +399,58 @@ def test_deploy_with_node_selection(self, db: Session, client: TestClient):
             expected_affinity=affinity,
         )
 
+    def test_validate_nuclio_version_compatibility(self):
+        mlconf.nuclio_version = "1.6.10"
+        assert not validate_nuclio_version_compatibility("1.6.11")
+        assert not validate_nuclio_version_compatibility("1.5.9", "1.6.11")
+        assert not validate_nuclio_version_compatibility("1.6.11", "1.5.9")
+        assert validate_nuclio_version_compatibility("1.6.9")
+        assert validate_nuclio_version_compatibility("1.5.9")
```

Comment:
Very good tests, please add another line here:
`assert validate_nuclio_version_compatibility("2.0.0")`

Reply:
I added `assert not validate_nuclio_version_compatibility("2.0.0")` and added comments for better flow understanding
(validate_nuclio_version_compatibility gets the minimum requirements)